helper/impls/helper-impl-110/src/main/java/com/linkedin/avroutil1/compatibility/avro110/FieldBuilder110.java

```diff
@@ -92,6 +94,34 @@ public FieldBuilder setOrder(Order order) {
     return this;
   }
 
+  @Override
+  public FieldBuilder addProp(String propName, String jsonObject) {
+    ObjectMapper objectMapper = new ObjectMapper();
+    try {
+      _props.put(propName, objectMapper.readTree(jsonObject));
```

Comment:
did you check that this is capable of reading "1" into an IntNode ?

Reply:
We want `"1"` to be read into an IntNode and `"\"1\""` to be read into a StringNode, correct?